### ISSUE TITLE
Add keyboard navigation short sentences

### ISSUE THREAD

**Issue by luifr10** [Orange-OpenSource/uuv#642]
Desired scenarios :
```gherkin
When I start keyboard navigation on the top of the page # Set keyboard focus at the top of the web page
Then the next keyboard element focused should be a button named {string} # Checks that the next item selectable by keyboard navigation is a button named "A button"
```

```gherkin
When I start keyboard navigation on the top of the page # Set keyboard focus at the top of the web page
Then the previous keyboard element focused should be a button named {string} # Checks that the previous item selectable by keyboard navigation is a button named "A button"
```

```gherkin
When I start keyboard navigation on the top of the page # Set keyboard focus at the top of the web page
Then the previous keyboard element focused should have name {string} and role {string}# Checks that the next item selectable by keyboard navigation is an element with role "role" and name "button"
```
